File: cs102/web-7/lab1.py
#!/usr/bin/python
print( "Content-type: text/html\n\n" )
print("<TITLE>Lab1</TITLE>")
print("<h1>Lab 1</h1>")
print( """
        <html>
        <body>
""" )
for x in range(100):
    print("<p>Line</p>")
print( """
        </body>
        </html>
""" )  

File: cs102/Week 7/Week 7 - Web/lab1.c
#include <stdio.h>
int main (int argc, char** argv) {
    fprintf(stdout,"Content-type: text/html\n\n");
    fprintf(stdout,"<TITLE>Lab 1</TITLE>\n");
    fprintf(stdout,"<h1>Lab 1</h1>\n");
    fprintf(stdout,"<html>\n");
    fprintf(stdout,"<body>\n");
    for (int x = 0; x<100; x=x+1) {
        fprintf(stdout,"<p>Line</p>\n");
    }
    fprintf(stdout,"</html>\n");
    fprintf(stdout,"</body>\n");
    return 0;
}  
